CRT.c

```diff
@@ -763,12 +763,15 @@ void CRT_init(int delay, int colorScheme) {
    setlocale(LC_CTYPE, "");
 
 #ifdef HAVE_LIBNCURSESW
-   if(strcmp(nl_langinfo(CODESET), "UTF-8") == 0) {
+   if (allowUnicode == true &&
```

Comment:
`allowUnicode == true` seems a bit redundant

Reply:
I can remove that if you prefer...